README.md

```diff
@@ -131,13 +131,18 @@ version: '3.4'
 
 services:
   wikibase:
-      image: "ghcr.io/mardi4nfdi/docker-wikibase:dev"
+    image: "ghcr.io/mardi4nfdi/docker-wikibase:dev"
     environment:
       XDEBUG_CONFIG: "remote_host=host.docker.internal"
+    expose:
+      - 9000
     volumes:
-     - ~/git/mediawiki/MathSearch:/var/www/html/extensions/MathSearch
+      - ./extensions-dev/<extension_to_debug>:/var/www/html/extensions/<extension_to_debug>
+      - ./debugging/php.ini:/usr/local/etc/php/php.ini
 ```
-Here `~/git/mediawiki/MathSearch` is the path of your local development checkout of the extension, you modify.
+Here `~/git/mediawiki/<extension_to_debug>` is the path of your local development checkout of the extension, you modify.
```

Comment:
path should be probably `./extensions-dev/...`

Reply:
zend installation is based on wikibase container setup, so it's non specfic.

Reply:
ok. in that case shouldnt debugging/php.ini be in docker-wikibase?

Reply:
it is loaded by docker compose, and can be customized (stuff like log-level ) this is easier to set in portal-compose